tests/integration/hive_metastore/test_workflows.py

```diff
@@ -97,3 +97,31 @@ def test_hiveserde_table_ctas_migration_job(ws, installation_ctx, prepare_tables
             assert ws.tables.get(f"{dst_schema.catalog_name}.{dst_schema.name}.{table.name}").name
         except NotFound:
             assert False, f"{table.name} not found in {dst_schema.catalog_name}.{dst_schema.name}"
+
+
+@pytest.mark.parametrize('prepare_tables_for_migration', ['regular'], indirect=True)
+def test_refresh_not_migrated_status_job(ws, installation_ctx, sql_backend, prepare_tables_for_migration, caplog):
+    tables, dst_schema = prepare_tables_for_migration
+    ctx = installation_ctx.replace(
+        extend_prompts={
+            r".*Do you want to update the existing installation?.*": 'yes',
+        },
+    )
+    ctx.workspace_installation.run()
+    second_table = list(tables.values())[1]
+    ctx.table_mapping.skip_table(dst_schema.name, second_table.name)
+    ctx.deployed_workflows.run_workflow("migrate-tables")
+    assert ctx.deployed_workflows.validate_step("migrate-tables")
+
+    remained_tables = list(
+        sql_backend.fetch(
```

Comment:
This is oke for now, but maybe we want to implement a fixture to get the query to make sure we test that, something like:

1. query = get_query("04_1_....sql")
2. Replace database
3. Run query
4. Verify results

Reply:
An issue was created. [#2396](https://github.com/databrickslabs/ucx/issues/2396)